app/src/components/Modals/add-user-modal.tsx

```diff
@@ -140,62 +148,79 @@ const AddUserModal: FC<AddUserModalProps> = ({ isOpen, onClose, userInfo }) => {
             justifyContent="center"
             fontWeight="semibold"
             fontSize="headline.sm"
+            fontFamily="heading"
             lineHeight="32"
             padding="24px"
-            borderBottom={"1px solid #fafafa"}
+            borderBottomWidth="1px"
+            borderStyle="solid"
+            borderColor="border.neutral"
           >
-            Add User
+            {t("add-user")}
           </ModalHeader>
           <ModalCloseButton marginTop="10px" />
-          <ModalBody>
+          <ModalBody p="24px" px="48px">
```

Comment:
I usually prefer using Chakra's unit system (which means numbers are 1/4 of the pixel value usually, similar to Tailwind).
So `p={6} px={12}` in this case. It's fine if you'd rather use pixels though.